### PR TITLE
adds WRF Alaska Downscaled Data from SNAP yaml file for AWS Open Data…

### PR DESCRIPTION
**Pull request by EarthScientist**
submission of the yaml file related toWRF-Alaska data from the Scenarios Network for Alaska + Arctic Planning (SNAP) at the University of Alaska Fairbanks (UAF).

### CODE REVIEW

File: datasets/wrf-alaska-snap.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Name: Historical and Projected Dynamically Downscaled Climate Data for the State of Alaska and surrounding regions at 20km spatial resolution and hourly temporal resolution
+Description: This dataset contains historical and projected dynamically downscaled climate data for the State of Alaska and surrounding regions at 20km spatial resolution and hourly temporal resolution. This data was produced using the Weather Research and Forecasting (WRF) model (Version 3.5). We downscaled both ERA-Interim historical reanalysis data (1979-2015) and both historical and projected runs from 2 GCM’s from the Coupled Model Inter-comparison Project 5 (CMIP5): GFDL-CM3 and NCAR-CCSM4 (historical run: 1970-2005 and RCP 8.5: 2006-2100).
+Documentation: http://ckan.snap.uaf.edu/dataset/historical-and-projected-dynamically-downscaled-climate-data-for-the-state-of-alaska-and-surrou
+Contact: http://directory.iarc.uaf.edu/peter-bieniek
+ManagedBy: Scenarios Network for Alaska + Arctic Planning at the International Arctic Research Center, University of Alaska, Fairbanks
+UpdateFrequency: as needed
+Tags:
+  - climate
```
Comment (jflasher) [inline datasets/wrf-alaska-snap.yaml]: Please add `aws-pds` as a tag.

File: datasets/wrf-alaska-snap.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - coastal
+  - earth observation
+  - environmental
+  - weather
+License: https://creativecommons.org/licenses/by/4.0/
+Resources:
```
Comment (jflasher) [inline datasets/wrf-alaska-snap.yaml]: Whenever you've got the S3 bucket created, please add it here.

File: datasets/wrf-alaska-snap.yaml (hunk 1 above)
Comment (EarthScientist) [reply datasets/wrf-alaska-snap.yaml]: added.

File: datasets/wrf-alaska-snap.yaml (hunk 2 above)
Comment (EarthScientist) [reply datasets/wrf-alaska-snap.yaml]: added.

**Comment by jedsundwall** [comment]
I've removed the empty DataAtWork fields and added quotes around the description to get it to render properly. Merging!

**Pull request merged**